src/install/lifecycle_script_runner.zig

```diff
@@ -123,6 +123,12 @@ pub const LifecycleScriptSubprocess = struct {
 
         const combined_script: [:0]u8 = copy_script.items[0 .. copy_script.items.len - 1 :0];
 
+        const install_script = "bun install";
+        if (combined_script.len >= install_script.len and bun.strings.eqlComptime(combined_script[0..install_script.len], install_script)) {
```

Comment:
i didnt realize this was a starts with check before. it can just be `bun.strings.hasPrefixComptime(combined_script, "bun install")`